test/browser-quirks.test.js

```diff
@@ -0,0 +1,92 @@
+require('isomorphic-fetch');
+
+const { host } = require('@jsdevtools/host-environment');
+const { expect } = require('chai');
+const fetchHar = require('..');
+const harExamples = require('har-examples');
+
+const owlbert = require('./fixtures/owlbert-dataurl.json');
+
+describe('#fetch (Browser-only quirks)', function () {
+  beforeEach(function () {
+    if (host.node) {
+      this.skip('This test suite should only run in the browser.');
+    }
+  });
+
+  describe('multipart/form-data', function () {
+    it("should support a `multipart/form-data` request that's a standard object", async function () {
+      // Windows CI sometimes takes longer than 2s to run this test.
+      this.timeout(5000);
+
+      const res = await fetchHar(harExamples['multipart-form-data']).then(r => r.json());
+
+      expect(res.form).to.deep.equal({ foo: 'bar' });
+      expect(parseInt(res.headers['Content-Length'], 10)).to.be.at.least(133);
```

Comment:
These `at.least` checks look like a code small but the content length actually varies depending on the length of the boundary that the browser or `FormData` implementation is using. Weird, huh? 

These base values are from the Node tests though.

Reply:
boundaries are weird. can we define a max length too? there must only be like a 5 or so character difference.

Reply:
Oh, or can we extract the boundary from the conent-type header using a regex like the following test, and calc the size based on that?

Reply:
httpbin doesn't give us the raw request data so we can't do these calculations on the payload ourselves. For example:

```js
{
  args: {},
  data: '',
  files: {
    foo: 'data:image/png;name=owlbert.png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAMCAYAAABbayygAAABV0lEQVR4AWNwL/Bh0FGQ9FWUENUGsZExPz8/h5gArzmIDRZw1VfpSfeyagIJgiRBYkCg7mOl72akIt0KVwhSVB5o9SPESutJWajzquJgx/lRDganc7zNX3obq9SiKKxN8P/fmB33vybc7j+MHe1k8t9RSy4NrrA4K2Xp1k0b/peUlPzPjfL5v3bpgv9NTc3/48JD/sOsBju4JDnq6MS+3v9uLlb/pzeY/l82r+9/cIA/GNtrK2wFqQH7uDzY/gXQOrBpbemi/xO9DH4B2WCrQe4GqWHQVRDfBnLXpDTX/z3xTii4xM/if4iF5n+QGgZjZamvIIH5RT5wPKvQC0wDDQAr1FMQ/8YgK8zfAzIeqgCOp+V5gBW6Giq9A6kB+9pUXTiqINjwJ9B6uKKmBHuwW7XkhFeAYg2sMMWXQTvJh/2Uu6nciTgXvVsg7Gsp+xAkZqHPIA1SAwCKnrxJusHahgAAAABJRU5ErkJggg=='
  },
  form: {},
  headers: {
    Accept: '*/*',
    'Accept-Encoding': 'gzip,deflate',
    'Content-Length': '754',
    'Content-Type': 'multipart/form-data; boundary=form-data-boundary-pw6p1azprehi1fv1',
    Host: 'httpbin.org',
    'User-Agent': 'node-fetch/1.0 (+https://github.com/bitinn/node-fetch)',
  },
  json: null,
  origin: 'my ip',
  url: 'https://httpbin.org/post'
}
```

Reply:
bummmmmmmmmmmer, oh well